### ISSUE TITLE
The correct syntax for using live file input is "<%= live_file_input @uploads.my_image %>". The previous sentence with "<.live_file_input @uploads.my_image />" is incorrect.

### ISSUE THREAD

**Issue by 0xmohsinpathan** [DockYard-Academy/curriculum#943]
**Describe the issue**
A clear and concise description of the issue.

**Reproduction Steps (If Applicable)**

**Expected behavior**

**Screenshots**

**Comment by BrooklinJazz** [comment]
This depends on the version of LiveView you're using.

LiveView 0.18 uses the following: https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#live_file_input/1

```elixir
<.live_file_input upload={@uploads.avatar} />
```

LiveView 0.17 uses the following: https://hexdocs.pm/phoenix_live_view/0.17.14/Phoenix.LiveView.Helpers.html#live_file_input/2

```elixir
<%= live_file_input @uploads.avatar %>
```

We do have an issue in the reading material where we seem to have examples of both. This will be resolved in the next release of the curriculum when we update all of the content for Phoenix 1.7.

Thank you for reporting! I assume this was the cause of the error you showed me today?

**Comment by BrooklinJazz** [comment]
solved in #941
